examples/runtime-instrumentation/Program.cs

```diff
@@ -22,7 +22,7 @@ public class Program
     public static void Main()
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
-            .AddRuntimeInstrumentation()
+            .AddProcessInstrumentation()
```

Comment:
Why update the runtime instrumentation example?

Reply:
it was a beautiful accident; reverted :innocent: